src/Tickets/Commerce/Gateways/Stripe/Payment_Intent_Handler.php

```diff
@@ -240,7 +240,7 @@ static function ( $item ) {
 		$metadata[ 'purchaser_email' ] = $order->purchaser_email;
 		$metadata[ 'event_name' ]      = get_post( current( $events_in_order ) )->post_title;
 		$metadata[ 'event_url' ]       = get_post_permalink( current( $events_in_order ) );
-		$metadata[ 'ticket_names' ]    =  implode( $ticket_names, ' ,' );
+		$metadata[ 'ticket_names' ]    = implode( ' ,', $ticket_names );
```

Comment:
Not related to your code, but not sure why we're wrapping the key with whitespaces, that should happen if there's a variable, not if we are writing the key.

Reply:
Not sure why, but those were from mine as well. Reverted now. Nice catch. Thanks